### DISCUSSION TITLE
Bring back the hover tabs, please.

### DISCUSSION THREAD

**Discussion by RaisTheCoder** [zen-browser/desktop#3266] [General]
This update was helpful for making YouTube playback issue get fixed with higher firefox base version, but now the side tabs is open all the time and I can't change it anymore, I checked everywhere and the options were deleted.
![image](https://github.com/user-attachments/assets/5f693821-0235-4867-8a82-b7417d5e3307)

**Comment by RaisTheCoder** [comment]
Hello, I started using compact mode and it seems doable, though I preferred something that's visible to me all the time, so I'll just hide the side bar and keep the address bar open. Thanks anyway <3

**Comment by vraravam** [comment]
With the 1.22 alpha version, this is now working.
Sidebar (without using compact mode) can collapse to only show the icons (so, always on with only icons), and then expand on hover

**Reply by RaisTheCoder** [reply]
Thanks a lot for the efforts! I got an update and when I hovered to tabs on compact mode, I noticed the tabs are collapsed
The sidebar also used to overlap with the top bar, that's also fixed too! thank you guys
